exercises/list_links.md

```diff
@@ -11,10 +11,11 @@ Start with an empty unordered html list in the static html:
 </ul>
 ```
 
-Now we can **dynamically** add your favourite links to the html list. 
-> 🔑 **Dynamically** means **capable of action or change**, when used in programming.
+Now we can **dynamically** add your favourite links to the html list.
 
-> The word **static**, in contrast with **dynamic**, means something that is **fixed**.
+> 🔑 **Dynamically** means **html elements can be updated by our blocks program**, in this context.
+
+> The word **static**, in contrast with **dynamic**, means **html elements are not updated by our blocks program**. The only way we can update the rendered HTML is to update the source HTML code itself.
```

Comment:
I think this is confusing: "The only way we can update the rendered HTML is to update the source HTML code itself." <- this is only true if we are artificially restricting ourselves to modifying the static html? It seems a bit circular

Reply:
The term "static"  is tricky to define for this purpose. To be honest, I'm not even sure what is gained by referring to "dynamic" vs "static" at this point. Sentences like this:
> Now we can **dynamically** add your favourite links to the html list.
I think we could dispense with the word **dynamically** here altogether - I don't think it adds anything of value to their understanding at this point. What do you think ?

Reply:
The reason the "dynamically" is there at all is to clarify that the point of the exercise is not to write a list of links in html (statically), but to do it in code/blocks (dynamically). Maybe it should just say that?